app/helpers/vita_min_form_builder.rb

```diff
@@ -75,6 +75,29 @@ def vita_min_select(
     html_output.html_safe
   end
 
+  def simplified_cfa_checkbox(method, label_text, options: {})
```

Comment:
this is identical to `CfaFormBuilder`'s `cfa_checkbox` except with the new class for styles, because you can't pass a class to `cfa_checkbox` to apply to the label. a classic example of honeycrisp not being very extendable :(

Reply:
I think now we have 2 simplified checkbox versions -- maybe it makes sense to create ONE that allows us to take modifier classes.